dev/buildchecker/main.go

```diff
@@ -379,10 +383,46 @@ func cmdHistory(ctx context.Context, flags *Flags, historyFlags *cmdHistoryFlags
 			if err != nil {
 				log.Fatal("Error storing OKAYHQ event okay.Push: ", err.Error())
 			}
+
 		}
+
 		if err := okayCli.Flush(); err != nil {
 			log.Fatal("Error posting to OKAYHQ okay.Flush: ", err.Error())
 		}
+
+	}
+
+	if historyFlags.slackReportWebHook != "" {
+
+		var totalBuilds int
+		var totalTime int
+		var totalFlakes int
+
+		for _, total := range totals {
+			totalBuilds += total
+		}
+
+		for _, incident := range incidents {
+			totalTime += incident
+		}
+
+		for _, flake := range flakes {
+			totalFlakes += flake
+		}
+
+		message := fmt.Sprintf(`
+		:bar_chart: Welcome to your weekly CI report for week ending: %s
```

Comment:
If we don't do this, we're prefixing the string with a newline, which is fine in the context of Slack, but can mess things up in other context. So it's a good habit to start straight after the backtick :) 

```suggestion
		message := fmt.Sprintf(`:bar_chart: Welcome to your weekly CI report for week ending: %s
```